### ISSUE TITLE
Multi monitor support

### ISSUE THREAD

**Issue by ghost** [stefantalpalaru/iccloader#1]
Hi,

I stumbled upon your program and I really like it. I use it every day. However, because I use multiple monitors, I disabled auto loading and created 4200K and 4201K profiles (one for each monitor), which I select manually from the right click menu. 

I understand this is a personal project and you might not need multi monitor support, so I'm planning to add it myself in the near future. If I succeed, I'll push it back for you to inspect and include, if you want to.

Regards,

Martijn

**Comment by stefantalpalaru** [comment]
You're right, I don't use it with multiple monitors, but I'm interested in the functionality if you can figure out how to target them individually using dispwin.
